### DISCUSSION TITLE
Decreasing IDs over time

### DISCUSSION THREAD

**Discussion by haianhle** [vdemichev/DiaNN#527] [General]
Hi Vadim, 

We've been having issue with our timsTOF, and it appears that upon a number of injections from the same well of Hela sample, the IDs go down over time. 

![image](https://user-images.githubusercontent.com/3183582/194751920-e3202c4b-25d0-479f-af0b-8f4c9057978f.png)
![image](https://user-images.githubusercontent.com/3183582/194751934-d6791f31-8020-42a8-a698-797905d4fee6.png)

While the MS1 signal from DIANN output does not exhibit this trend, the MS2 signal appears to go down too, which I guess makes sense because MS2 plays a  more important role in matching scores? Do you think this is normal and what do you think it point to in terms of the instrument's performance? 

![image](https://user-images.githubusercontent.com/3183582/194751974-57068b1d-c5bd-462a-acf0-7668850232f0.png)
![image](https://user-images.githubusercontent.com/3183582/194751980-f1b3bca8-0d0e-4d02-a635-7c37dc48ef25.png)

Thank you so much.

Best wishes,
Anh

**Comment by haianhle** [comment]
@vdemichev sorry for missing the label but the x-axis is the injection number.

**Reply by vdemichev** [reply]
This is not normal. Unless something is up with the sample vial, looks like a very significant sensitivity drop. But I am, not the best person to ask about this.
